app/Http/Controllers/MapController.php

```diff
@@ -16,10 +16,7 @@ class MapController extends Controller
      */
     public function show()
     {
-        //5 minutes caching
-        $places = Cache::remember('places-for-map', 60 * 5, function () {
-            return DB::table('places')->select('slug', 'long', 'lat')->where('is_approved', true)->get();
-        });
+        $places = DB::table('places')->select('slug', 'long', 'lat')->where('is_approved', true)->whereNotNull('long')->whereNotNull('lat')->get();
```

Comment:
Y'a une raison pour laquelle vous laissez tomber le cache? (Je demande, moi je cache quasiment toujours tout ! :P)

Reply:
To have up to date data since it wasn't affecting load times. I can add it back if you want.

Reply:
Nah, I thought it was because of possible API calls to save, all good! 👍